x-pack/plugins/security_solution/public/common/components/navigation/types.ts

```diff
@@ -28,7 +28,8 @@ export type UrlStateType =
   | 'cloud_posture'
   | 'findings'
   | 'entity_analytics'
-  | 'data_quality';
+  | 'data_quality'
+  | 'mitre_attack_coverage';
```

Comment:
It doesn't look to conform with `Coverage Overview` used for the page name. Maybe use `coverage_overview`?